bpf/tests/Makefile

```diff
@@ -50,6 +50,7 @@ clean:
 	rm -f $(wildcard *.d)
 
 BPF_TEST_FLAGS:= $(GO_TEST_FLAGS)
+BPF_BENCH_FLAGS:=
```

Comment:
Can we reuse `BENCHFLAGS` instead or as a basis for this variable? We could move the definition from `Makefile` to `Makefile.defs` for better sharing across different makefiles.

Reply:
Not sure, but I'll check.